internal/printer/printer_test.go

```diff
@@ -3277,6 +3277,16 @@ const c = '\''
 			source: `<html><body><h1>Hello world!</h1><style></style></body></html>`,
 			want:   []ASTNode{{Type: "element", Name: "html", Children: []ASTNode{{Type: "element", Name: "body", Children: []ASTNode{{Type: "element", Name: "h1", Children: []ASTNode{{Type: "text", Value: "Hello world!"}}}, {Type: "element", Name: "style"}}}}}},
 		},
+		{
+			name:   "element with unterminated double quote attribute",
+			source: `<main id="gotcha />`,
+			want:   []ASTNode{{Type: "element", Name: "main", Attributes: []ASTNode{{Type: "attribute", Kind: "quoted", Name: "id", Value: "", Raw: "\"g"}}}},
```

Comment:
I wonder if the `"g` in the `Raw` property is expected, I don't have context on why it would behave like this and don't want to assume that it's a bug

Reply:
Heh, I'd say that it's kinda partially expected, you can't have a valid raw value without the ending quote

Reply:
Yeah makes sense, I'll probably use a heuristic like stopping at the first non-alphabetic character. Waiting to get roasted 🫣